modules/client-configuration/pages/clients-debian.adoc

```diff
@@ -26,15 +26,15 @@ Bootstrapping can be used with {debian} clients for performing initial state run
 
 Some preparation is required before you can register {debian} clients to the {productname} Server:
 
-* If you are using Debian{nbsp}9, install the ``apt-transport-https`` package on the client before you attempt to register.
+* If you are using Debian{nbsp}9, install the required packages on the client before you attempt to register.
   On the client, at the command prompt, as root, run:
 +
 ----
-apt install apt-transport-https
+apt install apt-transport-https python-apt python3-apt
 ----
 
 * Ensure DNS is correctly configured and provides an entry for the client.
-    Alternatively, you can configure the ``/etc/hosts`` files on both the {productname} Server and the client with the appropriate entries.
+  Alternatively, you can configure the ``/etc/hosts`` files on both the {productname} Server and the client with the appropriate entries.
 * The client must have the date and time synchronized correctly with the {productname} Server before registration.
```

Comment:
```suggestion
* The client must have the date and time synchronized with the {productname} Server before registration.
```
drive-by edit.

Reply:
I think that exists in all the client files ...